Contribute/TOC.md

```diff
@@ -17,6 +17,12 @@
 ### [.NET docs style conventions](dotnet-style-guide.md)
 ### [Voice and tone guide](dotnet-voice-tone.md)
 ### [Pull request review process](dotnet-pr-review.md)
+##  [PowerShell docs](powershell\powershell-contribute.md)
+### [Contribution process](powershell\powershell-contribute-process.md)
```

Comment:
I think you should repeat the word powershell in these leaf items under PowerShell docs because the current TOC filter doesn't find them.